src/Form.js

```diff
@@ -101,17 +100,16 @@ class Form extends React.Component {
           />
         </label>
         <label>
-          {' '}
-          Your Last Name:{' '}
+          Last Name:
           <input
             name="userLastName"
             placeholder="Last Name"
             value={this.state.userLastName}
             onChange={this.handleChange}
           />
         </label>
-        <br />
-        <div class="headerSmall">Your address</div>
+
+        {/* <div className="headerSmall">Your address</div> */}
```

Comment:
Is this still needed?

Reply:
I guess not! I think I was leaving it there to remind me of another direction I wanted to go in the future, but I need to be more willing to trust that I will come up with good ideas again, and I'll likely have even better ideas next time.I'll take this bit out.